GMX-Bots/Large-Open-and-Increase-Positions/README.md

```diff
@@ -0,0 +1,56 @@
+# Large Open/Increase Position Size Bot
+
+## Description
+
+This bot detects when an open/increase position is large. The bot listens to `UpdatePosition` and `IncreasePosition` event emissions on GMX's Vault Contract and creates a finding if the position `size` exceeds the `threshold` 
+
```

Comment:
Suggesting an update for the following reasons:
1. first part is just for a better explanation. ( Feel free to improve it) 
2. We are not comparing the position size in both cases, so let's at least mention the correct parameter name used in comparison, ie `sizeDelta`. 

```suggestion
This bot detects a large size position opening and a large increase in an existing  position. It listens to `UpdatePosition` and `IncreasePosition` event emissions on GMX's Vault Contract and creates a finding if `sizeDelta` exceeds a `threshold`. 
```